package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "next",
-  "version": "3.0.1-beta.8",
+  "version": "3.0.1-beta.9",
```

Comment:
Sorry. It was my first external project contribution. 
I'm still learning the rules.